fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -749,6 +750,9 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VarAttr(name = ENABLE_SORT_AGGREGATE)
     private boolean enableSortAggregate = false;
 
+    @VarAttr(name = DISABLE_REWRITE_SUM_BY_ASSOCIATIVE_RULE)
+    private boolean disableRewriteSumByAssociativeRule = false;
```

Comment:
use enable instead of disable

Reply:
for high-cardinality group-by agg,  add another count agg functions, in the 1st AggregateStreamingOperator,  serialized overhead would increase significantly, should we testify these scenarios, if so ,we can prevent these rewrite.

Reply:
> use enable instead of disable

fixed

Reply:
> for high-cardinality group-by agg, add another count agg functions, in the 1st AggregateStreamingOperator, serialized overhead would increase significantly, should we testify these scenarios, if so ,we can prevent these rewrite.

I have add a restriction, for the queries with group by keys, if we can't reduce the number of agg functions, just skip rewriting